src/munge.spec.ts

```diff
@@ -31,6 +38,25 @@ const validateOfferCodecs = (offer: Sdp): boolean => {
   return true;
 };
 
+/**
+ * Check that the sdp offer contains rtcpFbValue or not.
+ *
+ * @param offer - The {@link Sdp} or {@link AvMediaDescription} to validate.
+ * @param rtcpFbValue - The rtcp-fb value to check.
+ * @returns True if the offer contains rtcp-fb value.
+ */
+const checkOfferContainsRtcpFeedback = (
+  offer: Sdp | AvMediaDescription,
+  rtcpFbValue: string
+): boolean => {
+  const mediaDescriptions = offer instanceof Sdp ? offer.avMedia : [offer];
+  return mediaDescriptions.some((av: AvMediaDescription) => {
+    return [...av.codecs.values()].some((ci: CodecInfo) => {
+      return ci.feedback.includes(rtcpFbValue);
+    });
+  });
```

Comment:
If you omit the curly braces, you don't need to use return here. So this can be:
```
  return mediaDescriptions.some((av: AvMediaDescription) =>
    [...av.codecs.values()].some((ci: CodecInfo) => ci.feedback.includes(rtcpFbValue))
  );
  ```

Reply:
I want to do this as well but I'll then get a eslint warning saying - 
Array.prototype.some() expects a return value from arrow function.eslint[array-callback-return](https://eslint.org/docs/rules/array-callback-return)
So as discussed, I'll just keep the return here.